index.d.ts

```diff
@@ -18,7 +18,7 @@ declare namespace unless {
          * and methods key-pairs. If the request path or path and method match, the middleware will not run. Check
          * Examples for usage.
          */
-        path?: string | string[];
+        path?: string | RegExp | (string | RegExp | { url: (string | RegExp); methods: string[]; })[];
```

Comment:
The docs don't make it clear, but from the implementation, if a path is passed that has a URL property, the same code is recursively called with the path set to that property instead. So the url member of the object can be any kind of object that could be passed unwrapped, including stupid things like `{ url: { url: { url: /regex/ } } }`

Again, because objects are just converted, you can mix them into the array.